2019-06-21 17:09:20 ERROR ControllerService:45 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:40)
	at com.revature.controller.Controller.launch(Controller.java:47)
	at com.revature.Main.main(Main.java:18)
2019-06-21 17:11:22 ERROR ControllerService:45 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:40)
	at com.revature.controller.Controller.launch(Controller.java:47)
	at com.revature.Main.main(Main.java:18)
2019-06-22 05:32:20 ERROR ControllerService:45 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:40)
	at com.revature.controller.Controller.<init>(Controller.java:46)
	at com.revature.Main.main(Main.java:17)
2019-06-22 05:37:38 ERROR ControllerService:45 - Could not login.
2019-06-22 06:07:43 ERROR ControllerService:51 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:44)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-22 06:13:02 ERROR ControllerService:50 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:43)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-23 19:57:36 ERROR ControllerService:50 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:43)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-23 20:27:46 ERROR ControllerService:50 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:43)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-24 01:47:40 ERROR ControllerService:55 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:48)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-24 02:25:18 ERROR ControllerService:126 - Could not set balance.
java.sql.SQLException: Could not commit with auto-commit set on
	at oracle.jdbc.driver.PhysicalConnection.commit(PhysicalConnection.java:4443)
	at oracle.jdbc.driver.PhysicalConnection.commit(PhysicalConnection.java:4490)
	at com.revature.service.ControllerService.depositMoney(ControllerService.java:123)
	at com.revature.controller.Controller.<init>(Controller.java:51)
	at com.revature.Main.main(Main.java:17)
2019-06-24 02:28:51 ERROR ControllerService:127 - Could not set balance.
java.sql.SQLException: Could not commit with auto-commit set on
	at oracle.jdbc.driver.PhysicalConnection.commit(PhysicalConnection.java:4443)
	at oracle.jdbc.driver.PhysicalConnection.commit(PhysicalConnection.java:4490)
	at com.revature.service.ControllerService.depositMoney(ControllerService.java:124)
	at com.revature.controller.Controller.<init>(Controller.java:51)
	at com.revature.Main.main(Main.java:17)
2019-06-24 15:16:17 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-24 16:10:19 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.junitTests.ControllerTest.loginTest(ControllerTest.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
2019-06-24 16:11:15 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.junitTests.ControllerTest.loginTest(ControllerTest.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
2019-06-24 16:13:23 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.junitTests.ControllerTest.loginTest(ControllerTest.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
2019-06-24 17:15:49 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.junitTests.ControllerTest.loginTest(ControllerTest.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
2019-06-25 09:58:14 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-25 10:32:48 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-25 10:34:58 TRACE ControllerService:48 - Entering login method with parameters: ljdlkasj, asljlkjd
2019-06-25 10:35:00 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-25 10:36:13 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-25 12:36:52 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 12:37:51 TRACE ControllerService:48 - Entering login method with parameters: saljlskfj, aljlsfj
2019-06-25 12:37:52 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-25 12:38:11 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-25 12:40:19 TRACE ControllerService:116 - Entering deposit method with parameters: balance = 150.0, deposit = 33.29
2019-06-25 12:40:19 TRACE ControllerService:118 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-25 12:40:19 INFO  ControllerService:124 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 183.29
2019-06-25 12:41:31 TRACE ControllerService:162 - Entering withdraw method with parameters: balance = 183.29, withdraw = 2.05
2019-06-25 12:41:31 TRACE ControllerService:164 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-25 12:41:31 INFO  ControllerService:170 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 181.23999999999998
2019-06-25 13:32:43 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 13:33:09 TRACE ControllerService:48 - Entering login method with parameters: dlkjskf, aljdlj
2019-06-25 13:33:11 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-25 13:37:53 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 13:38:08 TRACE ControllerService:48 - Entering login method with parameters: kljsd, sljlfs
2019-06-25 13:38:09 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:32)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.controller.Controller.<init>(Controller.java:40)
	at com.revature.Main.main(Main.java:17)
2019-06-25 13:43:57 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-25 13:49:46 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 13:50:21 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-25 13:51:19 TRACE ControllerService:116 - Entering deposit method with parameters: balance = 0.0, deposit = 50.0
2019-06-25 13:51:19 TRACE ControllerService:118 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-25 13:51:19 INFO  ControllerService:124 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 50.0
2019-06-25 13:53:05 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 13:53:18 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-25 13:53:37 TRACE ControllerService:116 - Entering deposit method with parameters: balance = 50.0, deposit = 50.0
2019-06-25 13:53:37 TRACE ControllerService:118 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-25 13:53:37 INFO  ControllerService:124 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 100.0
2019-06-25 13:53:58 TRACE ControllerService:164 - Entering withdraw method with parameters: balance = 100.0, withdraw = 50.0
2019-06-25 13:53:58 TRACE ControllerService:166 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-25 13:53:58 INFO  ControllerService:172 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 50.0
2019-06-25 14:09:09 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 14:09:15 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-25 14:09:35 TRACE ControllerService:116 - Entering deposit method with parameters: balance = 50.0, deposit = 0.0
2019-06-25 14:09:35 TRACE ControllerService:118 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-25 14:09:35 INFO  ControllerService:124 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 50.0
2019-06-25 14:13:25 TRACE ControllerService:164 - Entering withdraw method with parameters: balance = 50.0, withdraw = 0.0
2019-06-25 14:13:25 TRACE ControllerService:166 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-25 14:13:25 INFO  ControllerService:172 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 50.0
2019-06-25 17:25:22 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 17:25:22 TRACE ControllerService:208 - Entering register method
2019-06-25 17:25:55 TRACE ControllerService:208 - Entering register method
2019-06-25 17:26:45 TRACE ControllerService:208 - Entering register method
2019-06-25 17:27:21 TRACE ControllerService:219 - Getting connection with admin parameters
2019-06-25 17:27:21 INFO  ControllerService:230 - Creating user account with parameters: TEST_USER2, password
2019-06-25 17:27:21 ERROR ControllerService:247 - Could not create user account.
java.sql.SQLSyntaxErrorException: ORA-01935: missing user or role name

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.service.ControllerService.register(ControllerService.java:233)
	at com.revature.controller.Controller.<init>(Controller.java:57)
	at com.revature.Main.main(Main.java:17)
2019-06-25 17:32:44 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 17:32:44 TRACE ControllerService:208 - Entering register method
2019-06-25 17:33:03 TRACE ControllerService:219 - Getting connection with admin parameters
2019-06-25 17:33:03 INFO  ControllerService:230 - Creating user account with parameters: TEST_USER2, password
2019-06-25 17:33:03 ERROR ControllerService:247 - Could not create user account.
java.sql.SQLSyntaxErrorException: ORA-01935: missing user or role name

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1017)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:566)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:215)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:58)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1075)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3820)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:3897)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at com.revature.service.ControllerService.register(ControllerService.java:233)
	at com.revature.controller.Controller.<init>(Controller.java:57)
	at com.revature.Main.main(Main.java:17)
2019-06-25 17:35:32 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-25 17:35:32 TRACE ControllerService:208 - Entering register method
2019-06-25 17:36:58 TRACE ControllerService:219 - Getting connection with admin parameters
2019-06-25 17:37:33 INFO  ControllerService:230 - Creating user account with parameters: TEST_USER2, password
2019-06-26 06:04:36 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-26 06:04:36 TRACE ControllerService:208 - Entering register method
2019-06-26 06:04:49 TRACE ControllerService:219 - Getting connection with admin parameters
2019-06-26 06:04:51 INFO  ControllerService:245 - Creating user account with parameters: TEST_USER2, password
2019-06-26 06:04:51 INFO  ControllerService:252 - Granting admin options to created account (for purpose of testing)
2019-06-26 06:04:52 INFO  ControllerService:265 - Creating test bank account table
2019-06-26 06:09:09 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-26 06:09:36 TRACE ControllerService:116 - Entering deposit method with parameters: balance = 50.0, deposit = 50.0
2019-06-26 06:09:36 TRACE ControllerService:118 - Getting connection with user parameters: TEST_USER, p4ssw0rd
2019-06-26 06:09:36 INFO  ControllerService:124 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 100.0
2019-06-26 06:11:35 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER2, password
2019-06-26 06:21:05 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-26 06:21:17 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-26 06:21:30 TRACE ControllerService:208 - Entering register method
2019-06-26 06:21:42 TRACE ControllerService:219 - Getting connection with admin parameters
2019-06-26 06:21:42 INFO  ControllerService:225 - Creating user account with parameters: TEST_USER2, password
2019-06-26 06:21:44 INFO  ControllerService:232 - Granting admin options to created account (for purpose of testing)
2019-06-26 06:21:45 INFO  ControllerService:246 - Creating test bank account table
2019-06-26 06:22:27 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER2, password
2019-06-26 06:28:34 TRACE ControllerService:24 - Instantiation has been restricted.  REMOVE THIS LINE BEFORE FINALIZATION!
2019-06-26 06:28:56 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-26 06:30:09 TRACE ControllerService:208 - Entering register method
2019-06-26 06:31:17 TRACE ControllerService:219 - Getting connection with admin parameters
2019-06-26 06:31:18 INFO  ControllerService:225 - Creating user account with parameters: TEST_USER2, password
2019-06-26 06:31:21 INFO  ControllerService:232 - Granting admin options to created account (for purpose of testing)
2019-06-26 06:31:22 INFO  ControllerService:246 - Creating test bank account table
2019-06-26 06:32:06 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER2, password
2019-06-26 06:32:26 TRACE ControllerService:116 - Entering deposit method with parameters: balance = 0.0, deposit = 15.0
2019-06-26 06:32:26 TRACE ControllerService:118 - Getting connection with user parameters: TEST_USER2, password
2019-06-26 06:32:26 INFO  ControllerService:124 - REMOVE BEFORE FINALIZATION! Amount to be updated to: 15.0
2019-06-26 10:54:38 TRACE ControllerService:24 - Instantiation of ControllerService has been restricted.
2019-06-26 10:54:45 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER2, password
2019-06-26 10:54:47 ERROR ControllerService:56 - Could not login.
java.sql.SQLException: ORA-01017: invalid username/password; logon denied

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:392)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:385)
	at oracle.jdbc.driver.T4CTTIfun.processError(T4CTTIfun.java:938)
	at oracle.jdbc.driver.T4CTTIoauthenticate.processError(T4CTTIoauthenticate.java:480)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:655)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:249)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:416)
	at oracle.jdbc.driver.T4CTTIoauthenticate.doOAUTH(T4CTTIoauthenticate.java:825)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:596)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:715)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:385)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:30)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:564)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.revature.util.ConnectionUtil.getConnection(ConnectionUtil.java:17)
	at com.revature.service.ControllerService.login(ControllerService.java:49)
	at com.revature.controller.Controller.<init>(Controller.java:39)
	at com.revature.Main.main(Main.java:17)
2019-06-26 10:55:02 TRACE ControllerService:48 - Entering login method with parameters: TEST_USER, p4ssw0rd
2019-06-26 10:55:07 TRACE ControllerService:83 - Getting connection with user parameters: TEST_USER, p4ssw0rd
